src/classlibnative/inc/nlsinfo.h

```diff
@@ -145,53 +86,7 @@ class COMNlsInfo {
 
     static void QCALLTYPE nativeNormalizationInitNormalization(int NormForm, BYTE* pTableData);
```

Comment:
Actually, IsWindows8OrAbove is unconditionally returning true and we are not carrying the resources that would be required to make it work on pre-Win8 machines. We have shipped 1.0/1.1 that way. So all this is unreachable code.